### PR TITLE
Editing some typos and doubled words

### PR DESCRIPTION
**Pull request by aszasz**
Thank you for the language and the manual, it is an amazing work!
The markdown does not show line numbers as the pdf, so I removed reference to line number in one place, the rest are repeated words or repeated meaning. 
It should be only 9 changes, but 12 are seen... only by opening some of the files in vim, last lines are changed in a non-visible way. (I am new to all these things besides haxe: vim, git and github, so please forgive if that is inconvenient)

### CODE REVIEW

File: 04-class-field.tex [hunk 1]
```diff
@@ -60,7 +60,7 @@ \section{Property}
 
 \haxe{assets/Property.hx}
 
-For the most part, the syntax is similar to variable syntax, and the same rules indeed apply. Properties are identified by
+For the most part, the syntax is similar to variable syntax, and the same rules indeed apply. Properties are identified by:
```
Comment (Simn) [inline 04-class-field.tex]: Are you sure about that colon? I'm pretty sure I use this style (sentences which have listing in between) in several places. I'm also pretty sure I've seen it in other sources.

Comment (aszasz) [inline 04-class-field.tex]: No, I am not sure at all. I was just matching the style bellow, it makes 
total sense without it.

On 05-01-2015 19:03, Simon Krajewski wrote:
>
> In 04-class-field.tex 
> <https://github.com/HaxeFoundation/HaxeManual/pull/79#discussion-diff-22489336>:
>
> > @@ -60,7 +60,7 @@ \section{Property}
> >
> >  \haxe{assets/Property.hx}
> >
> > -For the most part, the syntax is similar to variable syntax, and the same rules indeed apply. Properties are identified by
> > +For the most part, the syntax is similar to variable syntax, and the same rules indeed apply. Properties are identified by:
>
> Are you sure about that colon? I'm pretty sure I use this style 
> (sentences which have listing in between) in several places. I'm also 
> pretty sure I've seen it in other sources.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/HaxeFoundation/HaxeManual/pull/79/files#r22489336>.
>

-- 
Arthur
arthur.szasz@gmail.com
(11) 9 6309 - 0246
(11) 3895 - 6753

Comment (Simn) [inline 04-class-field.tex]: Could you remove it? I'll merge afterwards.

Comment (aszasz) [inline 04-class-field.tex]: There you have it, thank you.

On 06-01-2015 08:41, Simon Krajewski wrote:
> In 04-class-field.tex
> <https://github.com/HaxeFoundation/HaxeManual/pull/79#discussion-diff-22517479>:
>
>> @@ -60,7 +60,7 @@ \section{Property}
>>
>>  \haxe{assets/Property.hx}
>>
>> -For the most part, the syntax is similar to variable syntax, and the same rules indeed apply. Properties are identified by
>> +For the most part, the syntax is similar to variable syntax, and the same rules indeed apply. Properties are identified by:
>
> Could you remove it? I'll merge afterwards.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HaxeFoundation/HaxeManual/pull/79/files#r22517479>.
>

-- 
Arthur
arthur.szasz@gmail.com
(11) 9 6309 - 0246
(11) 3895 - 6753

**Comment by Simn** [comment]
Thanks a lot for this!

**Pull request merged**